---
title: "Tumor progression in breast cancer reflected in gene expression patterns and mutational profile"
subtitle: "Project #5975"
author: "Vincent van Hoef"
date: today
title-block-banner: true
reference-location: margin
citation-location: margin
bibliography: ./assets/citations.bib
csl: ./assets/mla.cls
format:
    html:
        theme: cosmo
        toc: true
        toc-location: left
        css: "./assets/style.css"
        self-contained: false
        include-before-body: ./assets/custom.html
execute: 
  echo: false
  warning: false
comments: 
    hypothesis: true
---

```{r}
library(DT)
library(boxr)
box_auth()
```

:::{.callout-note collapse="true"}
## Click here to see the version and updates of this report

Here you can see the status of the report. This will be updated as sections are added or improved.

- Version 1: [12/5/22] Most results included, but sections on nanostring correlation and ESR1 mutation remain to be added.
- Version 1.1: [16/5/22] Update of TMB calculation; combinations of three variant callers and remove common variants + add paired plot.
- Version 1.2: [17/5/22] ESR1 mutation table added.
- Version 1.3: [17/5/22] Table and plot of correlation between Nanostring signature and TMB score added.

:::

# Introduction

This is the report of the analysis for project #5975, requested by Lisa Ryd√©n (LU). For this study, targeted next generation sequencing by hybridization capture was performed to get data about a panel of 442 genes. The main aim of the analysis is to call somatic variants in a collection of 205 breast cancer samples; some of which are matched pairs of primary tumors and distance metastases or primary tumors and lymph node metastases. Matched normal samples are however not available but might be included later for more accurate results. 

Contract:

* Preprocessing of 205 samples 
* CNV and SNV detection
* Format data to allow for analysis of correlation with survival and clinical annotation
* Detection of differences in CNV and SNV between primary tumors and lymph node or distant metastases
* Correlation with tumor grouping and BC360 gene signatures based on Nanostring expression data

[150 hours]

:::{.callout-note}
Please let me know if any of the visualizations or links seem to be erratic. We recently changed reporting tool and small issues might still be present.

You can leave comments on this report by opening the sidebar on the very right of this page. After signing up for Hypothes.is via the link at the top of the sidebar, commenting is quick and easy. Just highlight the section you want to remark upon and write your comment. Leave it in the public group ("Post to Public") and I will be able to see it and update the text if necessary.
:::

205 samples are included in this study. [The sample table contains 207 entries, but 2 samples were not sequenced.]{.aside} Sample 11, 12 and 19 seem corrupted and were removed because they cause the analysis pipeline to stop in error. I requested acces to the original files through Celemics but the files they send over are the exact same ones as the ones we have. Probably the actual sequencing facility will have to be contacted to get these samples from an earlier timepoint.

```{r}
meta <- box_read_csv("955662091441")
datatable(meta, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE))
```

# Material and Methods

This is a description of the methods used in this analysis. For a potential later manuscript M&M section, you can always contact NBIS for an up-to-date description of what ended up in the final manuscript, even after closing the project.

## Preprocessing

Raw reads were trimmed using TrimGalore v0.6.1 prior to mapping to GRCh37 using bwa mem of bwa v0.7.17. Subsequently, duplicates were tagged and removed using MarkDuplicates from picard v2.23.4. Base recalibration was performed and applied using GATK v4.2.0 with known sites datasets dbsnp_138, 1000G_phase1 and Mills_and_1000G_gold_Standard_indels.

## Variant Calling

Somatic variant calling of all samples was performed by combining the results of three variant callers; i.e. Mutect2 v4.2.0, VarScan v2.4.2 and freebayes v1.3.2. These were chosen because they are capable of running without a matched normal sample. Mutect2 was supplied with a generic panel-of-normals ("somatic-b37-Mutect2-exome-panel") and a germline resource ("gnomAD.r2.1.1.GRCh37.PASS.AC.AF.only"). VarScan was run with standard parameters and freebayes was run with -F 0.01 -C 5 --pooled-continuous and subsequently filtered to retain calls with QUAL > 20. Variant calling was restricted to the list of targets used for the hybrid capture. Variants found in all three callers were called again using the *call* command from bcftools 1.14 and annotated with VEP v99 using the *vcf2maf.pl* script from vcf2maf v1.6.21.

Additionally, somatic calling for the distant and lymph node metastases was done in a more rigorous manner. For these samples, the matched primary tumor was used as a *pseudo* normal sample and variant calling was performed by Mutect2 only. A custom panel-of-normals was created from the primary tumors using the *CreateSomaticPanelOfNormals* command from GATK. The same germline resource was used as the one employed for the primary tumor analysis. Contamination was estimated and Mutect2 calls were filtered as per the appropriate [GATK workflow](https://gatk.broadinstitute.org/hc/en-us/articles/360035531132). Keep in mind that this protocol only detects variants that are a variant in the metastatic sample but not in the primary tumor. It does not tell us anything about the variants of the primary tumor in itself; if both primary and metasasis have the variant it is not withheld in this analysis.

## Variant Filtering for Plotting

In addition to taking the intersect of the results of the three variant callers, only variants which are present at a variant allele frequency (VAF) < 1% in the Genome Aggregation Database (gnomAD) OR absent from gnomAD OR present in the COSMIC database AND with a gnomAD VAF < 1% were retained for plotting purposes. For plotting purposes, the R package MAFTools was used.

## Copy Number Analysis

For a rough estimation of the copy number alterations in the metastases, the R package pureCN was used [@riester_purecn:_2016]. It was designed for hybrid capture sequencing data, especially with medium-sized targeted gene panels without matching normal samples (matched whole-exome data is of course supported). The primary tumors were used to create a *pseudo* normal coverage profile which is then compared to the metastatic sample. Keep in mind that this means that any copy number alteration is a change in the metastasis as compared to the primary tumor. It does not tell us anything about the status of the primary tumor in itself.

:::{.callout-note}
Analysis of the copy number alterations in the primary tumors was considered to be too unreliable without a matched normal sample.
:::

## Tumor Mutational Burden

To calculate the TMB, the R package TMBLeR was used [@fancello_tmbler_2021]. The TMB is defined here as the number of mutations per Mb, excluding common variants (>1% in gnomAD) but including synonymous mutations. There are several ways to define and filter the TMB. The filtering used here is described as well in the TMBLeR's manual. The results of this analysis were correlated with the lymph node status and tumor location.

# Results

## Quality control 

Modern high throughput sequencers can generate hundreds of millions of sequences in a single run. Before analysing this sequences to draw biological conclusions you should always perform some simple quality control checks to ensure that the raw data looks good and there are no problems or biases in your data which may affect how you can usefully use it. Therefore, a quality control step was run on the raw reads before and after trimming by the tool [FastqQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/){target="_blank"}. Three samples are corrupted and were removed (11,12 and 19).

### FastQC

::: {.panel-tabset}

# Before Trimming

FastQC was run on the raw sequences as they are delivered by the sequencing facility so before the removal of adapter sequences and low quality bases. Read1 and Read2 are considered separately. Here we mainly look for outlier samples. Each plot has a "Help" buttong ginving some more info of the different quality assessments.

The full report can be downloaded [here](https://uppsala.box.com/s/hqc0smh7zhxs72pfq8syzchdotl7szv4) via the Download button in Box and visualized in any browser.

# After Trimming

The tool was run again after removal of low quality bases and adapter sequences.

The full report can be downloaded [here](https://uppsala.box.com/s/6m3aayzwcopwsiefi7cmu0nejbcfcdkk) via the Download button in Box and visualized in any browser.

:::

Overall, the quality of the samples and sequencing is sufficient to proceed with the downstream analysis.

### Coverage

The number of reads overlapping a specific nucleotide is an important quality metric because it will influence the confidence in the subsequent variant calls. Moreover, the more reads cover the target, the better we will be able to detect small clones and tumor heterogeneity. After mapping the trimmed reads to the genome, the average read depth per target (7455 targets in total) and per sample is calculated. The results per sample are shown in @fig-coverage-plot.

![Coverage per Sample Per Target](https://uppsala.box.com/shared/static/98qneihv5sxaowh83r6b5crpnuu9kplx.svg){#fig-coverage-plot width="100%" height="650px"}

Sample 224 and 28 have rather low coverage (22x and 33x respectively), but are kept in for the further downstream analysis. But good to keep in mind. The other samples have an average coverage beween 250x and 500x which is sufficient for somatic variant calling in heterogeneous tumors.

## Variant Calling

Here you can find the results of the variant calling analysis for the different sample groups.

### Primary Tumor

The primary tumors have been called by three different variant callers and the intersect of the SNPs of these three tools was retained for further analysis. The standard format to store variant information is as VCF files. In the Box window below you can see the VCF files for each of the three variant callers as well as the merged dataset. VCF files are not very user friendly and are mostly analyzed through different software packages, but they can be opened in Excel if necessary. 

<iframe src="https://uppsala.app.box.com/embed/s/qel9xgwg7brlcld3ckzxnynmsc2qhwol?sortColumn=date&view=list" style="width:100%; height:400px;"></iframe>

::: {.panel-tabset}

# Overview

Here you see a general overview of the variants in the Primary Tumor samples. It gives highlights the type and number of variants in the dataset as well as indicating the most commonly affected genes.

![Overview of Primary Tumor Samples](./Results/variant_calls_all/merged_calls/Plots/3_common/Distant_Tumor_Summary_3_common.svg){#fig-pt-overview width="100%" height="650px"}

# Oncoplot

An oncoplot gives more info on the 25 most commonly mutated genes. The PAM50 type of each sample is indicated by the bar at the bottom of the plot.

![Oncoplot of Primary Tumor Samples](./Results/variant_calls_all/merged_calls/Plots/3_common/oncoplots/Primary_Tumor_Oncoplot_3_common.svg){#fig-pt-oncoplot width="100%" height="650px"}

# Interaction

This plot will give you an indication whether the 25 most commonly mutated genes often occur in the same sample or if on the other hand they are mutually exclusive.

![Co-ocurrence of common mutations in Primary Tumor Samples](./Results/variant_calls_all/merged_calls/Plots/3_common/interactions/Primary_Tumor_Interaction_3_common.svg){#fig-pt-int width="100%" height="650px"}

:::

### Distant Metastases

The distant tumors have been called by Mutect2. In the Box window below you can see the VCF files obtained from the Mutect2 analysis (both raw and passing a quality filter). VCF files are not very user friendly and are mostly analyzed through different software packages, but they can be opened in Excel if necessary.

<iframe src="https://uppsala.app.box.com/embed/s/0t143y75tyaubwajjbbvjwj6f0r606y5?sortColumn=date&view=list" style="width:100%; height:400px;"></iframe>

::: {.panel-tabset}

# Overview

Here you see a general overview of the variants in the Distant Tumor samples. It highlights the type and number of variants in the dataset as well as indicating the most commonly affected genes.

![Overview of Distant Tumor Samples](./Results/distance_tumor/maf/plots/Summary_distance_tumor.svg){#fig-dt-overview width="100%" height="650px"}

# Oncoplot

An oncoplot gives more info on the 25 most commonly mutated genes. The PAM50 type of each sample is indicated by the bar at the bottom of the plot.

![Oncoplot of Distant Tumor Samples](./Results/distance_tumor/maf/plots/Oncoplot_distance_tumor.svg){#fig-dt-oncoplot width="100%" height="650px"}

# Interaction

This plot will give you an indication whether the 25 most commonly mutated genes often occur in the same sample or if on the other hand they are mutually exclusive.

![Co-ocurrence of common mutations in Distant Tumor Samples](./Results/distance_tumor/maf/plots/Interaction_distance_tumor.svg){#fig-dt-int width="100%" height="650px"}

:::

### Lymph Node Metastases

The lymph node samples have been called by Mutect2. In the Box window below you can see the VCF files obtained from the Mutect2 analysis (both raw and passing a quality filter). VCF files are not very user friendly and are mostly analyzed through different software packages, but they can be opened in Excel if necessary.

<iframe src="https://uppsala.app.box.com/embed/s/mst7xyg5uyi98zm2k9216nzaeuz1umtm?sortColumn=date&view=list" style="width:100%; height:400px;"></iframe>

::: {.panel-tabset}

# Overview

Here you see a general overview of the variants in the Lymph Node samples. It highlights the type and number of variants in the dataset as well as indicating the most commonly affected genes.

![Overview of Lymph Node Samples](./Results/lymph_node/maf/plots/Summary_lymph_node.svg){#fig-ln-overview width="100%" height="650px"}

# Oncoplot

An oncoplot gives more info on the 25 most commonly mutated genes. The PAM50 type of each sample is indicated by the bar at the bottom of the plot.

![Oncoplot of Lymph Node Samples](./Results/lymph_node/maf/plots/Oncoplot_lymph_node.svg){#fig-ln-oncoplot width="100%" height="650px"}

# Interaction

This plot will give you an indication whether the 25 most commonly mutated genes often occur in the same sample or if on the other hand they are mutually exclusive.

![Co-ocurrence of common mutations in Lymph Node Samples](./Results/lymph_node/maf/plots/Interaction_lymph_node.svg){#fig-ln-int width="100%" height="650px"}

:::

## Tumor Mutational Burden

The tumor mutational burden metric gives a general and rough indication of the amount of mutation in a specific tumor. It is defined as the number of mutations per Mb. The filtering strategy will influence this metric quite strongly but there is no consensus on a specific strategy. Here synonymous mutations have been kept, while common mutations have been filtered out, as is described in the software manual.

```{r}
tmb <- box_read_csv("955667979176")
tmb <- tmb[tmb$Filter == "vaf=0", !colnames(tmb) %in% "Design"]
datatable(tmb, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE, scrollX = TRUE))
```

### Correlation with tumor location

The TMB does seem to differ significantly between the different tumor locations. Distant tumors seem to have higher mutational burden than the lymph node samples, see @fig-tmb-loc for a plot of the three locations and t-tests between the groups (treated as unpaired data).

![TMB in the different tumor locations.](./Results/tmb/Tumor_Location_vs_TMB.svg){#fig-tmb-loc width="100%" height="650px"}

We can also try to take the paired nature of the data into account and use a paired Wilcoxon test to look for differances between the groups. Below you can see the plots for the different comaparisons, only looking at paired samples, see @fig-cor-paired.

::: {#fig-cor-paired height="800px"}

::: {.panel-tabset}

# PT_vs_LNM
![](./Results/tmb/Paired_PT_LNM.svg){width="100%" height="700px"}

# PT_vs_DM 
![](./Results/tmb/Paired_PT_DM.svg){width="100%" height="700px"}

# LNM_vs_DM 
![](./Results/tmb/Paired_LNM_DM.svg){width="100%" height="700px"}

:::
Visualization of TMB in paired samples.
:::

### Correlation with Lymph Node status

TMB does not seem to differ significantly between the lymph node status, see @fig-tmb-ln-stat.

![TMB according to Lymph Node status](./Results/tmb/Node_Status_vs_TMB.svg){#fig-tmb-ln-stat width="100%" height="650px"}

## Copy Number Alteration

Copy number alterations were estimated using the R package pureCN. In this analysis, metastases were compared to their matched primary tumor. It is important to keep in mind that therefore these results can be affected by the state of the primary tumor as well; e.g. an amplification in a lymph node tumor might indicate a deletion in the matched primary tumor rather than a genuine amplification.

### Distant Metastases

In below table you see an overview of the alteration in a specific gene and sample. You can look for specific genes by using the search field.

```{r}
dm_gene_no <- box_read_csv("956229121134")
datatable(dm_gene_no, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE, order = list(list(5, 'desc'))))
```

The full results of the pureCN analysis per patient can be downloaded in the window below. There are several files per patient here even though not all are equally informative. The most useful data is probably:

* The per gene statistics for each sample can for example be found in the per_genes folder. A description of the columns in these files can be found in Table 2 of [the pureCN manual](https://bioconductor.org/packages/release/bioc/vignettes/PureCN/inst/doc/PureCN.pdf).
* The alteration per chromosome and patient can be found in the chromosomes folder
* General info per sample can be found in the overview folder.

<iframe src="https://uppsala.app.box.com/embed/s/vfavpoydsc4814f7l0k5yemu1urvdx4c?sortColumn=date&view=list" style="width:100%; height:600px;"></iframe>

The 15 most commonly amplified or deleted genes are visualized in @fig-cna-dm.

::: {#fig-cna-dm height="800px"}

::: {.panel-tabset}

# Amplified
![](./Results/CNA/distant_tumor/Amplified_in_distant_tumor_vs_primary_tumor.pdf){width="100%" height="700px"}

# Deleted
![](./Results/CNA/distant_tumor/Deleted_in_distant_tumor_vs_primary_tumor.pdf){width="100%" height="700px"})

:::
Most Common Alterations in Distant Tumor versus Primary Tumor.
:::

### Lymph Node Metastases

In below table you see an overview of the alteration in a specific gene and sample. You can look for specific genes by using the search field.

```{r}
ln_gene_no <- box_read_csv("956229079317") 
datatable(ln_gene_no, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE, order = list(list(5, 'desc'))))
```

The full results of the pureCN analysis per patient can be downloaded in the window below. There are several files per patient here even though not all are equally informative. The most useful data is probably:

* The per gene statistics for each sample can for example be found in the per_genes folder. A description of the columns in these files can be found in Table 2 of [the pureCN manual](https://bioconductor.org/packages/release/bioc/vignettes/PureCN/inst/doc/PureCN.pdf).
* The alteration per chromosome and patient can be found in the chromosomes folder
* General info per sample can be found in the overview folder.

<iframe src="https://uppsala.app.box.com/embed/s/kkzcsx0nv9hdgz677mwe5jsd657h3fhw?sortColumn=date&view=list" style="width:100%; height:600px;"></iframe>

The 15 most commonly amplified or deleted genes are visualized in @fig-cna-lnm.

::: {#fig-cna-lnm height="800px"}

::: {.panel-tabset}

# Amplified
![](./Results/CNA/lymph_node/Amplified_in_lymph_node_vs_primary_tumor.pdf){width="100%" height="700px"}

# Deleted
![](./Results/CNA/lymph_node/Deleted_in_lymph_node_vs_primary_tumor.pdf){width="100%" height="700px"}

:::
Most Common Alterations in Lymph Node Sample versus Primary Tumor.
:::

## ER Mutations

Using the combination of the three variant callers, we can look for variants in ESR1. Because in the regular analysis as described above, variants in REC and LNM will be filtered out if they are also present in the primary tumor, we also ran all the variant callers on all samples. This way we can also detect ESR1 mutations in matched samples. All samples carrying a ESR1 mutation with a gnomAD frequency < 1% are shown in the table below.

```{r}
esr <- box_read_csv("958764984793") 
datatable(esr[,c(2:8)], rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE))
```

## Nanostring data

The TMB scores were correlated with the signature scores obtained using Nanostring. For each signature the pearson correlation coefficient with the TMB score was calculated. The results can be seen in the table below (table can be sorted to get most positive and negative correlating signature). Detailed description of the signatures is in the Nanostring report.

```{r}
cors <- box_read_csv("958821480662") 
cors$cor.coef <- gsub("c\\(cor = ", "", cors$cor.coef)
cors$cor.coef <- round(as.numeric(gsub(")", "", cors$cor.coef)), digits = 2)
datatable(cors[,c(2:4)], rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE))
```

Select a gene below to visualize the correlation.

```{r}
tmbl <- list.files(path = "./Results/tmb/TMB_Corr/", pattern = "_TMB_correlation.svg")
tmbl <- as.list(gsub("_TMB_correlation.svg", "", tmbl))
ojs_define(plots_tmb_corr = tmbl)
```

```{ojs}
viewof sig_tmb_corr = Inputs.select(plots_tmb_corr, { label: "Choose Signature to correlate: " })
html`<p></p>`
```

```{ojs}
//| label: fig-corr-tmb
//| fig-cap: "Correlation of Nanostring Signatures with the TMB score"
html`<img src="./Results/tmb/TMB_Corr/${sig_tmb_corr}_TMB_correlation.svg" width:100% height:600px></img>`
```


## Survival Data

An attempt was made to link the mutation data to the survival data. For this, samples were divided in two groups: samples with a mutation for a certain gene and another group of samples that are WT for the gene. The endpoint is Overall Survival. This was done within each sample group (i.e. primary, recurrence or lymph node).

::: {.panel-tabset}

### Primary Tumor

These are the results for the survival analysis within primary tumors only.

```{r}
pt_surv <- box_read_csv("956836277408") 
datatable(pt_surv, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE))
```

Select a gene below to visualize the Kaplan-Meier plot.

```{r}
fl <- list.files(path = "./Results/variant_calls_all/merged_calls/Plots/3_common/survival_analysis/", pattern = "_survival_curve_PT_3_common.svg")
fl <- as.list(gsub("_survival_curve_PT_3_common.svg", "", fl))
ojs_define(plots_surv_pt = fl)
```

```{ojs}
viewof genes_surv_pt = Inputs.select(plots_surv_pt, { label: "Choose Gene" })
html`<p></p>`
```

```{ojs}
//| label: fig-pt-surv
//| fig-cap: "Survival Plots Primary Tumor"
html`<img src="./Results/variant_calls_all/merged_calls/Plots/3_common/survival_analysis/${genes_surv_pt}_survival_curve_PT_3_common.svg" width:100% height:600px></img>`
```

### Distant Metastasis

These are the results for the survival analysis within distant metastasis only.

```{r}
dt_surv <- box_read_csv("956835681541") 
datatable(dt_surv, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE))
```

Select a gene below to visualize the Kaplan-Meier plot.

```{r}
fl <- list.files(path = "./Results/distance_tumor/maf/plots/", pattern = "_survival_curve_distance_tumor.svg")
fl <- as.list(gsub("_survival_curve_distance_tumor.svg", "", fl))
ojs_define(plots_surv_dt = fl)
```

```{ojs}
viewof genes_surv_dt = Inputs.select(plots_surv_dt, { label: "Choose Gene" })
html`<p></p>`
```

```{ojs}
//| label: fig-dt-surv
//| fig-cap: "Survival Plots Distant Tumor"
html`<img src="./Results/distance_tumor/maf/plots/${genes_surv_dt}_survival_curve_distance_tumor.svg" width:100% height:600px></img>`
```

### Lymph Node

These are the results for the survival analysis within lymph node samples only.

```{r}
ln_surv <- box_read_csv("956833214195") 
datatable(ln_surv, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE))
```

Select a gene below to visualize the Kaplan-Meier plot.

```{r}
fl <- list.files(path = "./Results/lymph_node/maf/plots/", pattern = "_survival_curve_lymph_node.svg")
fl <- as.list(gsub("_survival_curve_lymph_node.svg", "", fl))
ojs_define(plots_surv_ln = fl)
```

```{ojs}
viewof genes_surv_ln = Inputs.select(plots_surv_ln, { label: "Choose Gene" })
html`<p></p>`
```

```{ojs}
//| label: fig-ln-surv
//| fig-cap: "Survival Plots Lymph Node"
html`<img src="./Results/lymph_node/maf/plots/${genes_surv_ln}_survival_curve_lymph_node.svg" width:100% height:600px></img>`
```

:::

# Data Responsibility

- **NBIS & Uppnex** Unfortunately, we do not have resources to keep any files associated with the support request. We suggest that you safely store the results delivered by us. In addition, we ask that you remove the files from UPPMAX/UPPNEX after analysis is completed. The main storage at UPPNEX is optimized for high-speed and parallel access, which makes it expensive and not the right place for long time archiving.
- **Sensitive data** Please note that special considerations may apply to the human-derived sensitive personal data. These should be handled according to specific laws and regulations.
- **Long-term backup** The responsibility for data archiving lies with universities and we recommend asking your local IT for support with long-term data archiving. Also the newly established Data Office at SciLifeLab may be of help to discuss other options.

# Acknowledgements

If you are presenting the results in a paper, at a workshop or conference, we kindly ask you to acknowledge us.

- **NBIS Staff** are encouraged to be co-authors when this is merited in accordance to the ethical recommendations for authorship, e.g. [ICMJE recommendations](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html). If applicable, please include **Vincent van Hoef, National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Uppsala University**, as co-author. In other cases, NBIS would be grateful if support by us is acknowledged in publications according to this example: ["Support by NBIS (National Bioinformatics Infrastructure Sweden) is gratefully acknowledged"](https://www.nbis.se/resources/support.html).

- **UPPMAX** If your project has used HPC resources we kindly asks you to acknowledge UPPMAX and SNIC. If applicable, please add: ["The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project SNIC XXXX/Y-ZZZ"](https://www.uppmax.uu.se/support/faq/general-miscellaneous-faq/acknowledging-uppmax--snic--and-uppnex/).

- **NGI** In publications based on data from NGI Sweden, the authors must acknowledge SciLifeLab, NGI and UPPMAX: ["The authors would like to acknowledge support from Science for Life Laboratory, the National Genomics Infrastructure, NGI, and Uppmax for providing assistance in massive parallel sequencing and computational infrastructure"](https://ngisweden.scilifelab.se/info/faq#how-do-i-acknowledge-ngi-in-my-publication).

# Closing Procedures

You should soon be contacted by one of our managers, [Jessica Lindvall](jessica.lindvall@nbis.se) or [Henrik Lantz](henrik.lantz@nbis.se), with a request to close down the project in our internal system and for invoicing matters. If we do not hear from you within **30 days** the project will be automatically closed and invoice sent. Again, we would like to remind you about data responsibility and acknowledgements, see Data Responsibility and Acknowledgments sections.

You are naturally more than welcome to come back to us with further data analysis request at any time via [NBIS support](http://nbis.se/support/support.html).

Thank you for using NBIS and all the best for future research.