---
title: "Tumor progression in breast cancer reflected in gene expression patterns and mutational profile"
subtitle: "Project #5975"
author: "Vincent van Hoef"
date: today
title-block-banner: true
reference-location: margin
citation-location: margin
bibliography: ./assets/citations.bib
csl: ./assets/mla.cls
format:
    html:
        theme: cosmo
        toc: true
        toc-location: left
        css: "./assets/style.css"
        self-contained: false
        include-before-body: ./assets/custom.html
execute: 
  echo: false
  warning: false
comments: 
    hypothesis: true
---

```{r}
library(DT)
library(boxr)
box_auth()
```

:::{.callout-note collapse="true"}
## Click here to see the version and updates of this report

Here you can see the status of the report. This will be updated as sections are added or improved.

- Version 1: [12/5/22] Most results included, but sections on nanostring correlation and ESR1 mutation remain to be added.
- Version 1.1: [16/5/22] Update of TMB calculation; combinations of three variant callers and remove common variants + add paired plot.
- Version 1.2: [17/5/22] ESR1 mutation table added.
- Version 1.3: [17/5/22] Table and plot of correlation between Nanostring signature and TMB score added.

:::

# Introduction

This is the report of the analysis for project #5975, requested by Lisa Ryd√©n (LU). For this study, targeted next generation sequencing by hybridization capture was performed to get data about a panel of 442 genes. The main aim of the analysis is to call somatic variants in a collection of 205 breast cancer samples; some of which are matched pairs of primary tumors and distance metastases or primary tumors and lymph node metastases. Matched normal samples are however not available but might be included later for more accurate results. 

Contract:

* Preprocessing of 205 samples 
* CNV and SNV detection
* Format data to allow for analysis of correlation with survival and clinical annotation
* Detection of differences in CNV and SNV between primary tumors and lymph node or distant metastases
* Correlation with tumor grouping and BC360 gene signatures based on Nanostring expression data

[150 hours]

:::{.callout-note}
Please let me know if any of the visualizations or links seem to be erratic. We recently changed reporting tool and small issues might still be present.

You can leave comments on this report by opening the sidebar on the very right of this page. After signing up for Hypothes.is via the link at the top of the sidebar, commenting is quick and easy. Just highlight the section you want to remark upon and write your comment. Leave it in the public group ("Post to Public") and I will be able to see it and update the text if necessary.
:::

# Material and Methods

This is a description of the methods used in this analysis. For a potential later manuscript M&M section, you can always contact NBIS for an up-to-date description of what ended up in the final manuscript, even after closing the project.


# Results

## Quality control 

Modern high throughput sequencers can generate hundreds of millions of sequences in a single run. Before analysing this sequences to draw biological conclusions you should always perform some simple quality control checks to ensure that the raw data looks good and there are no problems or biases in your data which may affect how you can usefully use it. Therefore, a quality control step was run on the raw reads before and after trimming by the tool [FastqQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/){target="_blank"}. Three samples are corrupted and were removed (11,12 and 19).



# Data Responsibility

- **NBIS & Uppnex** Unfortunately, we do not have resources to keep any files associated with the support request. We suggest that you safely store the results delivered by us. In addition, we ask that you remove the files from UPPMAX/UPPNEX after analysis is completed. The main storage at UPPNEX is optimized for high-speed and parallel access, which makes it expensive and not the right place for long time archiving.
- **Sensitive data** Please note that special considerations may apply to the human-derived sensitive personal data. These should be handled according to specific laws and regulations.
- **Long-term backup** The responsibility for data archiving lies with universities and we recommend asking your local IT for support with long-term data archiving. Also the newly established Data Office at SciLifeLab may be of help to discuss other options.

# Acknowledgements

If you are presenting the results in a paper, at a workshop or conference, we kindly ask you to acknowledge us.

- **NBIS Staff** are encouraged to be co-authors when this is merited in accordance to the ethical recommendations for authorship, e.g. [ICMJE recommendations](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html). If applicable, please include **Vincent van Hoef, National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Uppsala University**, as co-author. In other cases, NBIS would be grateful if support by us is acknowledged in publications according to this example: ["Support by NBIS (National Bioinformatics Infrastructure Sweden) is gratefully acknowledged"](https://www.nbis.se/resources/support.html).

- **UPPMAX** If your project has used HPC resources we kindly asks you to acknowledge UPPMAX and SNIC. If applicable, please add: ["The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project SNIC XXXX/Y-ZZZ"](https://www.uppmax.uu.se/support/faq/general-miscellaneous-faq/acknowledging-uppmax--snic--and-uppnex/).

- **NGI** In publications based on data from NGI Sweden, the authors must acknowledge SciLifeLab, NGI and UPPMAX: ["The authors would like to acknowledge support from Science for Life Laboratory, the National Genomics Infrastructure, NGI, and Uppmax for providing assistance in massive parallel sequencing and computational infrastructure"](https://ngisweden.scilifelab.se/info/faq#how-do-i-acknowledge-ngi-in-my-publication).

# Closing Procedures

You should soon be contacted by one of our managers, [Jessica Lindvall](jessica.lindvall@nbis.se) or [Henrik Lantz](henrik.lantz@nbis.se), with a request to close down the project in our internal system and for invoicing matters. If we do not hear from you within **30 days** the project will be automatically closed and invoice sent. Again, we would like to remind you about data responsibility and acknowledgements, see Data Responsibility and Acknowledgments sections.

You are naturally more than welcome to come back to us with further data analysis request at any time via [NBIS support](http://nbis.se/support/support.html).

Thank you for using NBIS and all the best for future research.
